ffbs-parker-nodeconfig/files/usr/sbin/noderoute.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+# simple script to loop noderoute lua script
+
+tmpdir=/tmp/ff-Ohb0ba0u/
+
+LOGGER="logger -s -t noderoute.sh"
+$LOGGER Starting up.
+
+check_batman() {
+    # check if we have a functional batman-setup
+    if ip l | grep -q "bat0:"; then
+	# we have a bat0-interface. let's do some checks
+
+        if ip l | grep "bat0:" | grep -q DOWN; then
+            $LOGGER ERROR: BATMAN-interface is not up
+            return 1
+        fi
+
+        if ! ip l | grep "bat0:" | grep -q br-client; then
+            $LOGGER ERROR: bat0 not part of br-client
+            return 1
+        fi
+    else
+        $LOGGER ERROR: No BATMAN-interface found
+        return 1
+    fi
+    return 0
+}
+
+while true; do
+    $LOGGER Cycling
+    if ! lua /usr/share/lua/noderoute.lua $tmpdir | $LOGGER; then
+        $LOGGER noderoute.lua returned non-zero
+    fi
+    touch ${tmpdir}/noderoute-successful
+    sleep 10
+
+    if batctl gw | grep -q server; then
```

Comment:
Similar to the above, we can remove some nesting

```suggestion
    if ! batctl gw | grep -q server; then
        continue
    fi
```

Reply:
@SmithChart still open